### ISSUE TITLE
Spyder opens new instances when double-clicking Python scripts

### ISSUE THREAD

**Issue by jsh9** [spyder-ide/spyder#6899]
## Problem Description

I recently upgraded Spyder to 3.2.8 from 3.2.4.  I bind Spyder with any `.py` files on my Windows 10 computer, i.e., on the right-click menu of a certain `.py` file, under "open with", I choose "choose another app", and choose "C:\Anaconda3\Scripts\spyder.exe" to open `.py` files (and I check "Always use this app to open .py files" option).

After upgrading to 3.2.8, I found that every time I double-clicked a `.py` file, a new instance of Spyder opens up, with an empty cmd window pointing to "C:\Anaconda3\Scripts\spyder.exe" also open.

This was not the case for my previous versions of Spyder.

### What steps will reproduce the problem?

The steps are described above.

### What is the expected output? What do you see instead?

If a Spyder instance is already open, then double-clicking other `.py` files does not open new Spyder instances.  Instead, the `.py` files should be opened in the current Spyder instance.

### Paste Traceback/Error Below (if applicable)

N/A

## Package Versions

* Spyder: 3.2.8
* Python: 3.6.3
* Qt: 5.6.2
* PyQt: 5.6.0
* Operating System: Windows 10

### Dependencies

N/A

**Comment by ccordoba12** [comment]
This is supposed to be fixed. @dalthviz, could you take a look at this when you have some time? Thanks!

**Comment by ccordoba12** [comment]
See https://stackoverflow.com/q/33817046/438386 for ways people is using this feature on Windows.

**Comment by ccordoba12** [comment]
This is another related question: https://stackoverflow.com/q/50162165/438386

**Comment by dalthviz** [comment]
@jsh9 @jakubnyc could you help us testing the changes in #7260 to see if that solves the problem for you?

**Comment by jakubnyc** [comment]
@dalthviz   If there is a new version of Spyder, I will try it out and see if the problem has been solved.  The latest version I see is 3.2.8 which was not working.   I will download 3.2.9 when it is ready and give feedback.

**Comment by jsh9** [comment]
@dalthviz @ccordoba12 
I upgraded to Spyder 3.3.0.  On my Windows 10, here is what I observed:

(1) When no Spyder instance is opened, double-clicking a Python script opens up Spyder, but the cmd window stays open (I don't remember what the behavior of Spyder was 
(2) When there is a Spyder instance opened, double-clicking a Python script opens up that script inside Spyder, and the cmd window automatically closes

Maybe (1) is still a bug?  But the original problem associated with this issue (i.e., (2)) is solved.
